Directory.Packages.props

```diff
@@ -0,0 +1,131 @@
+    <PackageVersion Include="Microsoft.VisualStudio.ImageCatalog"                                    Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VisualStudio.Interop"                                         Version="17.4.0-preview-2-32826-307" />
+    <PackageVersion Include="Microsoft.VisualStudio.ManagedInterfaces"                               Version="8.0.50728" />
+    <PackageVersion Include="Microsoft.VisualStudio.RpcContracts"                                    Version="17.7.5-preview" />
+    <PackageVersion Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes"                           Version="15.0.36" />
+    <PackageVersion Include="Microsoft.VisualStudio.Settings.15.0"                                   Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VisualStudio.Setup.Configuration.Interop"                     Version="3.2.2146" />
+    <PackageVersion Include="Microsoft.VisualStudio.Shell.15.0"                                      Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VisualStudio.Shell.Design"                                    Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VisualStudio.Shell.Framework"                                 Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VisualStudio.Telemetry"                                       Version="17.7.47" />
+    <PackageVersion Include="Microsoft.VisualStudio.TemplateWizardInterface"                         Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VisualStudio.Threading"                                       Version="17.7.15-preview" />
+    <PackageVersion Include="Microsoft.VisualStudio.Threading.Analyzers"                             Version="17.7.15-preview" />
+    <PackageVersion Include="Microsoft.VisualStudio.Utilities"                                       Version="17.7.36307-preview.2" />
+    <PackageVersion Include="Microsoft.VisualStudio.Validation"                                      Version="17.6.11" />
+    <PackageVersion Include="Microsoft.VisualStudio.XmlEditor"                                       Version="17.3.32804.24" />
+    <PackageVersion Include="Microsoft.VSSDK.BuildTools"                                             Version="17.5.18-preview1" />
+    <PackageVersion Include="System.Threading.Tasks.Dataflow"                                        Version="7.0.0" />
+    <PackageVersion Include="Microsoft.VSDesigner"                                                   Version="17.0.0-preview-2-31223-026" />
+    <PackageVersion Include="VsWebSite.Interop"                                                      Version="16.8.30523.219" />
+
+    <!-- https://github.com/dotnet/roslyn/issues/53877 -->
+    <PackageVersion Include="StreamJsonRpc"                                                         Version="2.16.21-preview" />
```

Comment:
If this is not needed any more it can be removed.

Reply:
Local build passed, let's see if CI is happy with it.